src/bin/vip-import-sql.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { currentUserCanImportForApp, isSupportedApp } from 'lib/site-import/db-file-import';
+import { getFileMeta, uploadFile } from 'lib/client-file-uploader';
+
+/**
+ * - Include `import_in_progress` state & error out if appropriate (this likely needs to be exposed in the data graph)
+ * - Include `hasImporterS3Credentials` & error out if false (this needs to be implemented)
+ */
+const appQuery = `
+	id,
+	name,
+	organization { id, name },
+	environments{ id, appId, type, name, primaryDomain { name } }
+`;
+
+const err = message => {
+	console.log( chalk.red( message.toString().replace( /^(Error: )*/, 'Error: ' ) ) );
+	process.exit( 1 );
+};
+
+command( {
+	appContext: true,
+	appQuery,
+	requiredArgs: 1, // TODO print proper usage example
```

Comment:
Do you think it makes sense to add a confirmation step given this action will alter data?

Reply:
It definitely makes sense to add a confirm step.

I tried to specify `requireConfirm` in the `command` invocation, but it threw an error.  It looks pretty easy to fix, but will probably require a change to the lib.